### PR TITLE
Fix Warnings in Emby.Server.Implementations (Part 2)

### PR DESCRIPTION
**Pull request by barronpm**
**Changes**
Fixes more warnings (Down to 16 now). Depends on #6648 

**Issues**
Part of #2149

### CODE REVIEW

File: Emby.Server.Implementations/Library/UserViewManager.cs [hunk 1]
```diff
@@ -348,7 +348,8 @@ private IReadOnlyList<BaseItem> GetItemsForLatestItems(User user, LatestItemsQue
                 nameof(Year),
                 nameof(MusicGenre),
                 nameof(Genre)
-            } : Array.Empty<string>();
+            }
+                : Array.Empty<string>();
```
Comment (crobibero) [inline Emby.Server.Implementations/Library/UserViewManager.cs:352]: this indentation seems off?

**Review by crobibero — APPROVED** [review]

File: Emby.Server.Implementations/Library/UserViewManager.cs (hunk 1 above)
Comment (barronpm) [reply Emby.Server.Implementations/Library/UserViewManager.cs:352]: it's 4 spaces ¯\\\_(ツ)_/¯
cc @Bond-009

Comment (Bond-009) [reply Emby.Server.Implementations/Library/UserViewManager.cs:352]: The `?` on line 344 should be on the next line and indent the whole array

File: Emby.Server.Implementations/HttpServer/WebSocketConnection.cs [hunk 2]
```diff
@@ -150,8 +150,8 @@ public async Task ProcessAsync(CancellationToken cancellationToken = default)
                 {
                     await ProcessInternal(pipe.Reader).ConfigureAwait(false);
                 }
-            } while (
-                (_socket.State == WebSocketState.Open || _socket.State == WebSocketState.Connecting)
+            }
+            while ((_socket.State == WebSocketState.Open || _socket.State == WebSocketState.Connecting)
```
Comment (Bond-009) [inline Emby.Server.Implementations/HttpServer/WebSocketConnection.cs:154]: This one I dislike

**Review by Bond-009 — APPROVED** [review]

File: Emby.Server.Implementations/HttpServer/WebSocketConnection.cs (hunk 2 above)
Comment (barronpm) [reply Emby.Server.Implementations/HttpServer/WebSocketConnection.cs:154]: The analyzer warning, or just the specific way I changed it?

Comment (Bond-009) [reply Emby.Server.Implementations/HttpServer/WebSocketConnection.cs:154]: The warning, imo the while should be on the same line

File: Emby.Server.Implementations/Library/UserViewManager.cs (hunk 1 above)
Comment (barronpm) [reply Emby.Server.Implementations/Library/UserViewManager.cs]: I think that my last commit fixes this

**Pull request merged**